spec/requests/api/v1/apps_spec.rb

```diff
@@ -87,8 +125,67 @@
       end
     end
 
-    context 'without required params' do
-      let(:client_name)   { '' }
+    context 'with multiple redirect_uris as a string' do
+      let(:redirect_uris) { "https://redirect1.example/\napp://redirect2.example/" }
```

Comment:
This is previously accepted string input, and would result in multiple redirect_uri's being stored / usable.